content/en/cloud_cost_management/tags/_index.md

```diff
@@ -0,0 +1,126 @@
+  text: "Tag Explorer"
+- link: "/getting_started/tagging/"
+  tag: "Documentation"
+  text: "Getting Started with Tags"
+---
+
+## Overview
+
+Tags help you investigate and understand your cloud and SaaS costs across any dimensions. Tags are made of tag keys and values (ex: in `aws_product:ec2`, the tag key is `aws_product`, and the value is `ec2`.)
+
+Cloud Cost Management automatically enriches your cost data with tags from multiple sources, to help you achieve higher cost allocation and get deeper insight into who owns infrastructure costs in your ever changing cloud environments. Using tags, you can allocate shared costs fairly, create accurate reports, and track costs by team, service, or environment. 
+
+## Where tags come from
+
+Across all cloud and SaaS providers, Datadog collects tags from the following sources (including enriching cost data with data from other Datadog products) and adds them to your cost data:
+
+| Source | What tags are collected | Description |
+|---|---|---|
+| All Providers | Bill columns | Such as AWS Cost and Usage Report (CUR) columns, Google Billing Export columns, etc |
+| Datadog Enrichment | Host Agent | Tags added to host metadata by the Datadog agent running on the host |
+| Datadog Enrichment | Service Catalog | Tags associated with this service in the APM Service Catalog |
+| Datadog Enrichment | Integration Tiles | Tags added to the Datadog integration tile for a specific cloud account. Integration tile tags apply to all costs in that account. Requires enabling the provider integration for each account |
+| Datadog Enrichment | Data Observability | Tags from the Datadog’s Data Observability product, requires enabling BigQuery monitoring |
+| Datadog Enrichment | Cloud Network Monitoring | Source and destination dimensions from the Datadog Cloud Network product. Requires enabling Cloud Network in the Datadog agent |
+| Kubernetes Enrichment | Kubernetes Node | User-defined tags found on Kubernetes nodes monitored with Datadog | 
+| Kubernetes Enrichment | Kubernetes Pod | User-defined tags found on Kubernetes pods monitored with Datadog | 
+| Kubernetes Enrichment | Kubernetes Persistent Volume | User-defined tags found on Persistent Volumes in Kubernetes clusters monitored with Datadog |
+| Kubernetes Enrichment | Kubernetes Persistent Volume Claim | User-defined tags found on Persistent Volume Claims in Kubernetes clusters monitored with Datadog | 
+| Cloud Cost Management | Cloud Cost Aliases | Some aliases are tags derived from provider cost data, used to simplify the cost data model, such as `aws_product`, which is an alias of `lineItem/ProductCode`. Additional tags that exist on both cost data and integration metrics are added, allowing cost data and usage data to be combined in Custom Allocation Rules, dashboards, and notebooks |
+| Cloud Cost Management | Cloud Cost Allocation | Tags created during cost allocation that specify the split of shared resources, such as `allocated_spend_type` |
```

Comment:
maybe a link to [the docs](https://docs.datadoghq.com/cloud_cost_management/cost_allocation/container_cost_allocation/?tab=aws#understanding-spend)